src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
 spring.application.name=mini-survey-monkey
+spring.datasource.url=jdbc:h2:file:/Users/ali/Documents/SYSC4806/mini-survey-monkey/src/main/resources/Data/SYSC4806;AUTO_SERVER=true
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=Banana
```

Comment:
Do you think it would make sense to keep the username and password secret? Doesn't matter too much, but we could inject these as secrets with GitHub actions.